components/ui/button.tsx

```diff
@@ -51,38 +52,56 @@ export const buttonClasses = cva(
       disabled: {
         true: "opacity-50",
       },
+      fullWidth: {
+        true: "w-full justify-center",
+      },
     },
     defaultVariants: {
       disabled: false,
-      variant: "filled",
+      fullWidth: false,
       size: "medium",
+      variant: "filled",
     },
   },
 );
 
-const Button = ({
-  children,
-  className,
-  disabled,
-  onClick,
-  prefix,
-  size,
-  suffix,
-  type = "button",
-  variant,
-}: ButtonProps) => {
-  return (
-    <button
-      className={buttonClasses({ className, disabled, size, variant })}
-      disabled={disabled}
-      onClick={onClick}
-      type={type}
-    >
-      {prefix}
-      {children}
-      {suffix}
-    </button>
-  );
-};
+const Button = React.forwardRef(
```

Comment:
Wrapped component in forwardRef so headlessui's `Menu.Item` component (and later others if necessary) can pass ref to DOM `button` element.